content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -36,10 +36,10 @@ If `startingDeadlineSeconds` is set to a large value or left unset (the default)
 and if `concurrencyPolicy` is set to `Allow`, the jobs will always run
 at least once.
 
-Jobs may fail to run if the CronJob controller is not running or broken for a
-span of time from before the start time of the CronJob to start time plus
-`startingDeadlineSeconds`, or if the span covers multiple start times and
-`concurrencyPolicy` does not allow concurrency.
+For every CronJob, the CronJob controller checks how many schedules it missed in the duration from its last scheduled time till now. If there are more than 100 missed schedules, then it doesn't start the job. This would mean that, for example, If `concurrencyPolicy` does not allow concurrency, a job will be counted as missed if it was attempted to be scheduled when there was a previously scheduled cronjob still running.
+
+It is important to note, that if the field `startingDeadlineSeconds` is set (not `nil`), it will count how many missed jobs occurred from the value of `startingDeadlineSeconds` till now. For example, if `startingDeadlineSeconds` is `200`, It will count how many missed jobs occurred in the last 200 seconds. 
```

Comment:
when you say "count" do you mean "count as Failed"?

Reply:
@ericat If a cronjob was to be scheduled at a certain point in time, but the controller didn't schedule it for some reason. I explained it a bit more here: f4f5f03